### DISCUSSION TITLE
Lines added/deleted doesn't seem to be accurate

### DISCUSSION THREAD

**Discussion by developomp** [lowlighter/metrics#916] [Q&A] [answered]
**Topic:** [lines plugin](https://github.com/lowlighter/metrics/tree/master/source/plugins/lines)

### The problem:

I set up the metrics to run every 24 hour for my [GitHub profile](https://github.com/developomp). Everything seems to be working fine, but I noticed that the lines statistics could fluctuate by hundreds of thousands of lines every day.

Here's an example: [Feb 25](https://github.com/developomp/developomp/blob/674b77668be39664451a91a21b5374d190f5e796/github-metrics.svg) | [Feb 26](https://github.com/developomp/developomp/blob/eb2db9e1a0a347757917dd4be895ee89f060d349/github-metrics.svg) | [Feb 27](https://github.com/developomp/developomp/blob/555f98e377f774bf6012fd1af82840e5c1dce3e5/github-metrics.svg) | [Feb 28](https://github.com/developomp/developomp/blob/ee3da0f2518e8dbf6e0d07a3c7e479933a2ebd70/github-metrics.svg)

### Possible problem and solution:

I was looking at the plugin [logic](https://github.com/lowlighter/metrics/blob/master/source/plugins/lines/index.mjs) and the [mock data](https://github.com/lowlighter/metrics/blob/master/tests/mocks/api/github/rest/repos/getContributorsStats.mjs) used to test the plugin and noticed that aside from lines added and removed, there's also data for lines changed. Including this in the infographic might help.

**Comment by lowlighter** [comment, marked as answer]
Hi 👋 !
Yes, we could display the changed lines too, as you mentionned it'd wouldn't be too hard.

I'm not sure how GitHub actually compute these but it's possible that it's a scheduled job that do it which may reclassify the added/deleted into changed from time to time but it's all gueses.

The plugin doesn't really do post-processing on these number, apart from summing them together so I don't think it's an issue with it directly
